website/src/components/NotificationBanner/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, makeStyles } from '@material-ui/core';
+import notifications from '../../resources/notifications.json';
```

Comment:
Since only one notification is supported at a time, can we name the file notification.json?